scripts/mkdist.js

```diff
@@ -14,43 +14,40 @@ function mkdist (opts, cb) {
 
   var dist = opts.dist
   var ver = dist.version
-  if (ver.match(/^v/)) ver = ver.substring(1)
```

Comment:
For now, that is the build process for Orbit. When we are in a more stable process, we will move beyond `master`. 

@haadcode might have more to say on this matter (or he can link to somewhere where we've had this discussion before).